### PR TITLE
testbench: Remove uneeded variable

### PR DESCRIPTION
**Pull request by PayalKshirsagar**
Remove unneeded temporary local variable, its declaration and newlines.

### CODE REVIEW

File: tools/testbench/ll_schedule.c [hunk 1]
```diff
@@ -12,10 +12,6 @@ int schedule_task_init_ll(struct task *task, uint32_t uid, uint16_t type,
 			  enum task_state (*run)(void *data), void *data,
 			  uint16_t core, uint32_t flags)
 {
-	int ret;
```
Comment (dbaluta) [inline tools/testbench/ll_schedule.c]: Lets use a prefix for commit subject e.g: ```testbenc: Remove uneeded variable```

Comment (PayalKshirsagar) [reply tools/testbench/ll_schedule.c]: Okay, did it.

Comment (dbaluta) [reply tools/testbench/ll_schedule.c]: @PayalKshirsagar I was referring to the commit message not the Pull Request. Just use git commit --amend in your setup, modify the commit and then use --force to push the branch. (git push -f )

File: src/arch/xtensa/hal/interrupts.c [hunk 2]
```diff
@@ -415,9 +415,9 @@ XtHalVoidFunc *Xthal_tram_trigger_fn = xthal_null_func;
 unsigned xthal_vpri_to_intlevel(unsigned vpri)
 {
```
Comment (dbaluta) [inline src/arch/xtensa/hal/interrupts.c]: @PayalKshirsagar please don't touch this code. Is taken from xtensa and we are not changing the coding style here.

Comment (PayalKshirsagar) [reply src/arch/xtensa/hal/interrupts.c]: Okay, Thank you:)

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
@PayalKshirsagar should this be closed ?

**Comment by PayalKshirsagar** [comment]
> @PayalKshirsagar should this be closed ?

I didn't close it, I also wondered.
It has done automatically.

**Comment by dbaluta** [comment]
@PayalKshirsagar Looks like this PR has 0 commits. Maybe you pushed the wrong branch :).

Anyhow, I suggest you to take it slow. Lets take it gradually. 

1) Do not send newer patches until the existing ones are accepted.
2) Work with reviewers to fix their comments.

**Comment by PayalKshirsagar** [comment]
> @PayalKshirsagar Looks like this PR has 0 commits. Maybe you pushed the wrong branch :).
> 
> Anyhow, I suggest you to take it slow. Lets take it gradually.
> 
>     1. Do not send newer patches until the existing ones are accepted.
> 
>     2. Work with reviewers to fix their comments.
> 
> 
> I appreciate the enthusiasm.

Thank you for your guidance:)
Can I send the 2 commits of this PR in two different PRs?

**Comment by dbaluta** [comment]
Something went terribly wrong with this PR as it has 0 commits now :). So, lets leave it closed and open another one.

And, yes lets go with two different PR. But only just send the first PR and then wait to be merged then send the second one.

Later after you get better acquianted with github stuff we can do more PR in parallel, and PRs with multiple commits.

**Pull request closed without merging**